subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -74,20 +76,29 @@ struct pgps_index {
 	uint32_t storage_extent;
 	int store_block;
 
-	/* array of pointers to predictions, in sorted time order */
+	/* Array of memory offsets to predictions, in sorted time order.
+	 * If flash device is external, this must be passed
+	 * to read_prediction() to read a copy to a local buffer.
+	 * If flash device is internal, it can be converted directly to
+	 * a pointer.
+	 */
 	struct nrf_cloud_pgps_prediction *predictions[NUM_PREDICTIONS];
```

Comment:
The only time the predictions are in RAM are during the download process from the cloud, or when using external flash and a given prediction is cached in RAM.  Otherwise, they are always in flash -- either the onboard flash or external flash.

Reply:
I'm not sure I want to change this, but I'll think about it.

Reply:
Ah, yeah, not sure why I thought they were in RAM. Local memory would have been a better term